### DISCUSSION TITLE
Filter monsters by XP

### DISCUSSION THREAD

**Discussion by SansCB** [5e-bits/5e-srd-api#575] [Q&A]
Hello! I am trying to filter with an API call all the monsters that have an XP value, but it returns all the monsters. I'm doing it like this:
https://www.dnd5eapi.co/api/monsters?xp=200

Testing with other indexes, such as challenge_rating, it does work:
https://www.dnd5eapi.co/api/monsters?challenge_rating=10

I don't know if this is a bug, or I'm doing something wrong using the API.

**Comment by fergcb** [comment]
I don't think we have an XP filter for monsters, but in D&D 5e, XP is a function of Challenge Rating (CR, the `challenge_rating` field). That is, all monsters with a challenge rating of 1 give 200xp, CR 2 always gives 450xp, etc.

So instead, you can filter by CR like you're already doing. So for monsters that give 200xp, you can do the following:

```
https://dnd5eapi.co/api/monsters?challenge_rating=1
```

Note that for CR ½ and ¼ you should use the values `0.5` and `0.25` respectively.
